src/content/docs/features/stronghold.mdx

```diff
@@ -3,12 +3,232 @@ title: Stronghold
 description: Encrypted, secure database.
 ---
 
-import Stub from '@components/Stub.astro';
 import PluginLinks from '@components/PluginLinks.astro';
+import { Tabs, TabItem } from '@astrojs/starlight/components';
+import CommandTabs from '@components/CommandTabs.astro';
 
 <PluginLinks plugin="stronghold" />
 
-<Stub>
-	Based on
-	https://github.com/tauri-apps/plugins-workspace/tree/v2/plugins/stronghold
-</Stub>
+Store secrets and keys using the [IOTA Stronghold](https://github.com/iotaledger/stronghold.rs) encrypted database and secure runtime.
+
+## Supported Platforms
+
+- Windows
+- Linux
+- macOS
+
+## Setup
+
+_This plugin requires a Rust version of at least **1.75**_
+
+Install the stronghold plugin to get started.
+
+<Tabs>
+  <TabItem label="Automatic" >
+
+Use your project's package manager to add the dependency:
+
+{ ' ' }
+
+<CommandTabs
+  npm="npm run tauri add stronghold"
+  yarn="yarn run tauri add stronghold"
+  pnpm="pnpm tauri add stronghold"
+  cargo="cargo tauri add stronghold"
+/>
+
+  </TabItem>
+  <TabItem label = "Manual">
+
+1. Install the stronghold plugin by adding the following to your `Cargo.toml` file:
```

Comment:
Would you mind using the steps component for this part?

Reply:
Sure! It looks better 👍